esp-hal/src/dma/mod.rs

```diff
@@ -2011,6 +2047,9 @@ pub trait RegisterAccess: crate::private::Sealed {
     /// Set the address of the first descriptor.
     fn set_link_addr(&self, address: u32);
 
+    /// Enable/disable outlink-writeback
+    fn set_auto_wrback(&self, enable: bool);
```

Comment:
This method should have a nice name.
```suggestion
    fn set_auto_write_back(&self, enable: bool);
```